SemanticSearch2.md

```diff
@@ -0,0 +1,320 @@
+Semantic Search API V2
+=====================
+
+The semantic search API supports two endpoints: /query and /document.
+The /query endpoint interprets and provides related entities for queries.
+The /document endpoint parses a document, either a job or resume, and returns the most relevant semantic entities.
+
+# Table of Contents
+_____________
+
+## Query Endpoint
+
+- [Query Description](#query-description)
+
+- [Query Request Information](#query-request-information)
+
+- [Query Response](#query-response)
+
+## Document Endpoint
+
+- [Document Description](#document-description)
+
+- [Document Request Information](#document-request-information)
+
+- [Document Response](#document-response)
+
+## General Information
+
+- [Error Handling](#error-handling)
+
+# Query Description
+
+Interprets the meaning of a user's query. Used to parse a query into the intended phrases, to identify the type of entities being searched for (job title, skill, company, location, etc.), and to identify the most related other phrases/entities that help better represent the intended query. This service also applies stored user overrides to queries when user parameters are supplied in conjunction with the Semantic Overrides API. 
+
+
+# Query Request Information
+
+
+HTTP method: GET or POST
+Parameters (query/form):
+-	query (required) : query to parse and from which to infer meaning.
+-	language (optional) : two letter language code followed by underscore, followed by two letter country code. Determines the language in which enrichments are returned. Required to use personalized overrides. Defaults to en_us. Currently, allowed values are en_us, en_gb, fr_fr, de_de, and nl_nl.
+-	user_id (optional) : user id for which to look up personalized overrides. Defaults to null. 
+-	relationships_threshold : the mimimum weight of a relationship entity to be added as enrichments. Can be used to prune the result size. Defaults to 0.5. Allowed values are any number between 0 and 1.
+ 
+Example 2.0 request: 
+```
+HTTP GET
+Accept: application/json;version=2.0
+https://api.careerbuilder.com/core/semanticsearch/query?query=registered%20nurse&language=en_us&user_id=U1234
+```
+
+# Query Response
+
+The query response is divided into two parts. First is the parsed_input node, which gives information about parsing of extracted keywords in the query (this node is missing in the document response).
+Second is the extracted_keywords node, which gives the type and related entities of each extracted keyword. Enrichments can be overridden to be returned with a selected=true flag for a given user_id through the Overrides API.
+
+2.0 response:
```

Comment:
We need to add additional information besides a sample response. I left a comment in Carsten's pr of documentation. Ill repost it here. We need to explain more about the fields that appeared in the response. There are two kinds of extracted keyword entities, one kind is from solr, one kind is from PAN. we need to document them here. Ruben, you can ask @carschno for a list of PAN entity enrichment.